src/utils/sqs_consumer.ts

```diff
@@ -0,0 +1,91 @@
+import { SQS } from 'aws-sdk';
+import delay from 'delay';
+
+import { ONE_SECOND_MS } from '../constants';
+import { logger } from '../logger';
+
+import { SqsClient } from './sqs_client';
+
+export type MessageHandler = (message: SQS.Types.Message) => Promise<any>;
+
+export class SqsConsumer {
```

Comment:
One small thing: can the SQS Consumer accept a `uid` parameter? `uid` can be a unique indentifier of this wallet (so this can be set to the account address)? Ideally, this parameter is passed in to every log command, so that we can diagnose issues relevant to a particular account